src/main/java/uk/gov/justice/digital/client/s3/S3DataProvider.java

```diff
@@ -31,7 +31,7 @@ public S3DataProvider(JobArguments arguments) {
 
     public Dataset<Row> getSourceData(SparkSession sparkSession, String schemaName, String tableName) {
         String tablePath = tablePath(arguments.getRawS3Path(), schemaName, tableName);
-        String fileGlobPath = ensureEndsWithSlash(tablePath) + arguments.getBatchLoadFileGlobPattern();
+        String fileGlobPath = ensureEndsWithSlash(tablePath) + arguments.getCdcFileGlobPattern();
```

Comment:
This was a mistake in the batch job PR - the whole class should have been included in this PR instead with the CDC file glob pattern.